hpycc/get.py

```diff
@@ -256,8 +257,8 @@ def get_logical_file(*args, **kwargs):
                       "instead.")
 
 
-def get_thor_file(connection, thor_file, max_workers=15, chunk_size=10000,
-                  max_attempts=3, max_sleep=10, dtype=None):
+def get_thor_file(connection, thor_file, max_workers=10, chunk_size='auto', max_attempts=3, max_sleep=60,
+                  min_sleep=50, dtype=None):
```

Comment:
what happens if we don't deal with min sleep here. I don't really like it and I'm not sure I want to introduce it. Can we create  a new issue/ pull request for it?

Reply:
When I say here, I mean in this pull request

Reply:
Removed. min_sleep is no longer passed and is instead
`math.ceil(max_sleep*0.75)`